llvmlite/tests/test_binding.py

```diff
@@ -632,6 +632,7 @@ def test_emit_assembly(self):
         target_machine = self.target_machine()
         mod = self.module()
         ee = self.jit(mod, target_machine)
+        target_machine.set_verbose(True)
```

Comment:
The exact structure of the assembly comments, including the character that starts the comment, is dependent on the selected target, so I'm not sure how. Besides there isn't much to break, except the function not being called correctly...